tutorials/hyperparameter_tuning_with_ray/tune_model.ipynb

```diff
@@ -631,12 +631,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.10"
+   "version": "3.9.7"
   },
   "orig_nbformat": 4,
   "vscode": {
    "interpreter": {
-    "hash": "07dd84f3285103b45d36665747e055dbb0ef78c9d17293fd1b33b99205a40b05"
+    "hash": "5859d33511df864b0b7226a715510a0165ef032ed4b83eb4ae2c092f0788759c"
```

Comment:
vat is this thing???

Reply:
Hmmm, I don't really know. It looks like something that the Jupyter notebook is maintaining internally.